flixel/FlxSprite.hx

```diff
@@ -207,8 +207,8 @@ class FlxSprite extends FlxObject
 	public var bakedRotationAngle(default, null):Float = 0;
 
 	/**
-	 * Set alpha to a number between `0` and `1` to change the opacity of the sprite.
-	 @see https://snippets.haxeflixel.com/sprites/alpha/
+		* Set alpha to a number between `0` and `1` to change the opacity of the sprite.
+		@see https://snippets.haxeflixel.com/sprites/alpha/
```

Comment:
lets fix the indenting on this
```suggestion
	* Set alpha to a number between `0` and `1` to change the opacity of the sprite.
	* @see https://snippets.haxeflixel.com/sprites/alpha/
```

Reply:
the indenting here was from an auto formatter running everytime I made changes to be honest